src/bundle/Resources/config/routing.yml

```diff
@@ -336,6 +336,14 @@ ezplatform.content_type.list:
     requirements:
         contentTypeGroupId: \d+
 
+ezplatform.content_type.draft_list:
+    path: /contenttypegroup/{contentTypeGroupId}/contenttype/draftlist
```

Comment:
Up for discussion: Do we really want to copy over other `/contenttypegroup` paths which are clearly badly defined? For readability we should use `-` dash character to separate words.